### ISSUE TITLE
Black Armory Triumphs

### ISSUE THREAD

**Issue by t3rr11** [Bungie-net/api#802]
So i've been waiting to see some black armory triumphs to arrive but there was on 5-6 that i can see. I use these to track how many completions people have in my clan then rank by the most. 

Eg. Last wish completions, Ep completions, Blind wells completed, Levi's completed. Etc. 

I was hoping with Black Armory there would be just as many but there is no way to track how many Volundr forges completed. I'm sure more triumphs will come but i was wondering if i could put in a request to have more of these tracking triumphs. 

I know i can track completions though the emblem, though that requires O-Auth and scanning everyone in a clan wouldn't be viable for rankings unless all 100 people in the clan allow my bot to scan their profiles. 

Triumphs are great for this, Forsaken had so many things to track and this update came with 5 possibly 6 triumphs and none of which would be good for rankings.

**Comment by t3rr11** [comment]
Yeah, I'd like to be able to track more things though triumphs, I could track Volundr completions though the emblem. "Might of volundr" But since i don't use O-Auth and when tracking an entire clan if not everyone has allowed my application, It gets different to make rankings. 
But since triumphs don't require O-Auth it makes them perfect for clan rankings.

I really love the addition of Triumphs and having them track specific things. Since this post i have seen some good ones to track like "Scourge of the past completions" Which i was worried wouldn't be there since black armory had so few triumphs.

**Comment by ckhicks** [comment]
Side note: that is a really cool idea to track clan stats purely through Triumphs...now you've got me 🤔as well!
